core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -1633,11 +1633,28 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 global = globals + global_idx;
                 global_addr = get_global_addr(global_data, global);
                 aux_stack_top = *(uint32 *)(frame_sp - 1);
-                if (aux_stack_top <= exec_env->aux_stack_boundary.boundary) {
+                if (
+/* clang-format off */
+#if WASM_ENABLE_MULTI_MODULE != 0
+                    aux_stack_top <= module->module->aux_stack_bottom
+                                         - module->module->aux_stack_size
```

Comment:
Had better not use aux_stack_size directly, in multi-thread, the aux stack is divided into many sub areas according to the setting of max thread num.

Reply:
SO, which is better? 

Multiple modules in varients `exec_env` and stacks or multiple modules in one `exec_env`

Reply:
My opinion is the first one is better, refer to the patch I uploaded, just like calling a normal wasm function from native/host.